project1/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cranes wife</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Abel|IM+Fell+French+Canon+SC|Notable&display=swap"
+        rel="stylesheet">
+
+</head>
+
+<body>
+    <hr />
```

Comment:
Per the convo in class, the `hr` element should be used to represent a shift in topic, separating out sections of text.

```suggestion
```

Here, instead, what you want to use is the CSS `border` property on your `h1`. This is an example of what that would look like: https://codepen.io/angeliquejw/pen/PoYvdMJ?editors=0100